apps/meteor/tests/end-to-end/api/01-users.js

```diff
@@ -751,6 +751,91 @@ describe('[Users]', function () {
 		});
 	});
 
+	describe('[/users.setAvatarFromService]', () => {
+		let user;
+		let userCredentials;
+		before(async () => {
+			user = await createUser();
+			userCredentials = await login(user.username, password);
+		});
+		before((done) => {
+			updateSetting('Accounts_AllowUserAvatarChange', true).then(() => {
+				updatePermission('edit-other-user-avatar', ['admin', 'user']).then(done);
+			});
+		});
```

Comment:
please use `async/await` (just like the lines above):
```suggestion
		before(async () => {
			await updateSetting('Accounts_AllowUserAvatarChange', true);
			await updatePermission('edit-other-user-avatar', ['admin', 'user']);
		});
```